aperturedb/Connector.py

```diff
@@ -380,8 +380,9 @@ def _query(self, query, blob_array = [], try_resume=True):
                 if self._send_msg(data):
                     response = self._recv_msg()
                     if response is not None:
-                        querRes = queryMessage_pb2.queryMessage()
-                        querRes.ParseFromString(response)
+                        querRes = queryMessage.queryMessage()
+                        querRes.ParseFromString(response if
```

Comment:
How about making a Parse and Build method in queryMessage, so that all the details of dealing with querymessage are contained in one place.

Reply:
that seems like a good idea.

Reply:
added.